tests_integration/nativeapp/test_events.py

```diff
@@ -43,3 +43,23 @@ def test_app_events_cant_specify_first_and_last(runner, temporary_working_direct
         )
         assert result.exit_code == 1, result.output
         assert "--first and --last cannot be used together." in result.output
+
+
+@pytest.mark.integration
+def test_app_events_reject_invalid_type(runner, temporary_working_directory):
+    project_name = "myapp"
+    result = runner.invoke_json(
+        ["app", "init", project_name],
+        env=TEST_ENV,
+    )
+    assert result.exit_code == 0, result.output
+
+    with pushd(Path(os.getcwd(), project_name)):
```

Comment:
You can use the test data directory instead of running `snow app init`. V2 coverage would also be appreciated!

```suggestion
@pytest.mark.integration
@enable_definition_v2_feature_flag
@pytest.mark.parametrize("test_project", ["napp_init_v1", "napp_init_v2"])
def test_app_events_reject_invalid_type(test_project, runner):
    with project_directory(test_project):
```

Reply:
fixed this one and the other one in the same file